docs/api-testing/composer.md

```diff
@@ -2,82 +2,601 @@
-    * Leave the rest of the fields blank and click **Confirm changes** (checkmark icon).
+5. On the **Compose** tab, make sure the **Input** and **Visual** views are toggled on.
 
-:::tip
-For detailed information, see [Assertion Components](/api-testing/composer/assertion-components/).
+<img src={useBaseUrl('/img/api-testing/input-visual-on.png')} alt="Input and Visual settings on" width="500"/>
+
+:::note
+You can use either the **Visual** composer (guides you through building components, with no coding required) or the **Code** composer (requires you to write code from scratch). For this guide, we're using **Visual**.
 :::
 
-### Run Test
-14. Before you run the test, select the **Save** icon.<img src={useBaseUrl('img/api-fortress/2021/01/save.png')} alt="save icon"/> at the top of the Composer.
-15. Then select the **Run** icon <img src={useBaseUrl('img/api-fortress/2021/01/run.png')} alt="run test icon"/> directly next to the Save icon.<img src={useBaseUrl('img/api-fortress/2021/01/composerToolbar.png')} alt="Test Composer Tool Bar"/>
+For more information, see [Input Sets](/api-testing/composer/#input-sets) and [Visual View and Code View](/api-testing/composer/#visual-view-and-code-view).
+
+### Edit a Test
+To edit a test at any time, on the **Projects** page, on the **Tests** tab, hover over a test name and then click **Edit Test**.
+
+<img src={useBaseUrl('/img/api-testing/edit-test-nav.png')} alt="Navigating to the test editor" width="300"/>
+
+### Add Test Components
+When test components are combined, they act as our test logic. There are three component types available in API Testing:
+  * [I/O Request Test Components](/api-testing/composer/io-components)
+  * [Assertion Test Components](/api-testing/composer/assertion-components/)
+  * [Logical Test Components](/api-testing/composer/logical-components/)
+
+#### Add an I/O Request Test Component
+To create a simple `GET` request and validate that response is correct:
+  1. In API Testing, on the **Compose** page, click the **Add component** button.
+
+  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen" width="600"/>
+
+  2. In the list of component options, click the **GET** component.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-nav.png')} alt="Navigating to the GET request window" width="400"/>
+
+  3. In the **GET request** window, in the **Url** field, enter **ht<span>tps://</span>api.us-west-1.saucelabs.com/rest/v1/public/tunnels/info/versions**.
+
+  This URL will return a json response body.
+  4. In the **Variable** field, enter **payload**. This variable stores the response.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-window.png')} alt="Editing in the GET request window"/>
+
+  5. Leave the rest of the fields blank and then click the **Confirm changes** icon.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-confirm-changes.png')} alt="Confirm changes icon" width="200"/>
```

Comment:
@LoryApiFortress If you want to have a discussion offline about screenshot standards and guidelines, please set something up. Otherwise, I ask that you trust me to do my job.